mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -171,8 +171,13 @@ msgstr "{user} is leeching health from {target}."
 
 msgid "combat_state_overfeed_get"
 
-msgstr "{user} Overfed {target}. {target}'s speed was halved, {target}'s hp was fully restored"
+msgstr "{user} Overfed {target}. {target} 's speed was halved, {target}' s hp was fully restored"
 
+msgid "combat_amnesia_get"
+msgstr "{user} 's dodge fell. \n {user} 's attack damage types rose."
```

Comment:
Maybe the "attribute x increased/decreased/restored" could be separate strings, so that we only have to translate them once.

Reply:
im not sure i did it right, please take a look